operators/pkg/instctrl/cloudinit.go

```diff
@@ -86,6 +90,41 @@ func (r *InstanceReconciler) EnforceCloudInitSecret(ctx context.Context) error {
 	return nil
 }
 
+// Get the specs of the volume
+func (r *InstanceReconciler) GetNfsSpecs(ctx context.Context) (serverIp, serverName, path string, err error) {
+	instance := clctx.InstanceFrom(ctx)
+	secretName := types.NamespacedName{Namespace: instance.Namespace, Name: UserPvcSecret}
+
+	secret := corev1.Secret{}
+	if err = r.Get(ctx, secretName, &secret); err != nil {
+		ctrl.LoggerFrom(ctx).Error(err, "failed to retrieve secret", "secret", secretName)
+		return
+	}
+
+	var ok bool
```

Comment:
As above, you should be able to define local variables on the go using the `:=` assignment, avoiding to define them beforehand (referenced mostly to `var ok bool` in this case).

Reply:
Ok, remove useless declarations